### PR TITLE
Et 35 identity

### PR DESCRIPTION
**Pull request by jimentwood**

### CODE REVIEW

**Review by alisonhofer — APPROVED** [review]
Looks good!

File: source/help/registerhelp.md [hunk 1]
```diff
@@ -1,48 +1,46 @@
...
-and password. Your user name must be unique and cannot be changed.  
-Recommendations for arXiv password:
 
--  8 characters or more in length
+Your username must be unique and cannot be changed.  
 
-- May contain upper/lowercase letters, numbers, and one or more underscores (_) - one underscore is required
+arXiv registration also asks for your name and your Organization. If you are unaffiliated you may enter "Indepdendent" as your organization.
+
+Select which archives you are most likely to submit to. This will filter the category options available if you submit an article in the future.
```
Comment (rebamrich) [inline source/help/registerhelp.md:9]: Suggestion: Remove "in the future". I think it filters options immediately after successful account creation.

File: source/help/registerhelp.md [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
 
-Second, we will ask you for some personal information such as your name,
-your institution (or organization) and which archives you are most
-likely to submit to. We will then verify the email for your account.
+Account password requirements:
 
-*Policy Notes:*
+-  8 characters or more in length
+
+- May contain upper/lowercase letters, numbers, and one or more underscores (_) - one underscore is required
```
Comment (rebamrich) [inline source/help/registerhelp.md:22]: Suggestion: Separate this into two parts and spell out "uppercase". 

- May contain uppercase and lowercase letters, and numbers
- Must contain at least one underscore (_)

File: source/help/registerhelp.md [hunk 3]
```diff
@@ -1,48 +1,46 @@
...
--   It is a violation of our policies to misrepresent your identity or
-    organizational affiliation. Claimed affiliation should be current in
-    the conventional sense: e.g., physical presence, funding, e-mail
-    address, etc. Misrepresentation of identity or affiliation, for any
-    reason, is possible grounds for immediate and permanent suspension.
--   Each user may have only one registered arXiv account. If you are
-    attempting to register a new account and are unsure if you already
-    have an account please contact [arXiv user support](https://arxiv.org/support) .
--   Users must comply with arXiv's [Code of conduct](policies/code_of_conduct.md).
+- May not contain any other special characters besides underscore - no other punctuation or symbols are allowed (ex. $, ., -,@)
```
Comment (rebamrich) [inline source/help/registerhelp.md:24]: Suggestion: Make this two sentences. The hyphen-as-dash is confusing in a phrase about special characters. Also suggest changing "ex." to "e.g.". "Ex." doesn't hold standard meaning.   

- May not contain any other special characters besides underscore. No other punctuation or symbols are allowed (e.g., $, ., -, @)

File: source/help/registerhelp.md [hunk 4]
```diff
@@ -1,48 +1,46 @@
...
 
-#### Password
+Some successful password examples could be: “Pass_Word_123” or “pAssWoRd_123”
```
Comment (rebamrich) [inline source/help/registerhelp.md:26]: Suggestion: Reword to "A successful password format could be:".

File: source/help/registerhelp.md [hunk 5]
```diff
@@ -1,48 +1,46 @@
...
 password can be changed from your [personal information page](https://arxiv.org/user/).
 If you forget your password, you can [recover your
 password](https://arxiv.org/user/lost_password) using your registered e-mail address.
 
-#### <span id="emailchange"></span>Email
+<span id="emailchange"></span>
+
+## Email
 
 If you need to change your preferred e-mail address, for example because
```
Comment (rebamrich) [inline source/help/registerhelp.md:37]: Change the four instances of "e-mail" in this paragraph to "email" to follow other instances on this page (and CMoS).

File: source/help/registerhelp.md [hunk 6]
```diff
@@ -1,48 +1,46 @@
...
 you have changed institutions, then you should visit the [e-mail change
 form](https://arxiv.org/auth/email-change-form). This will ask for your new e-mail
 address and will send an e-mail with a verification code to your new
-address to make sure that your new address is correct. Please allow up to an hour
+address to make sure that your new address is correct. 
+
+Please allow up to an hour
 for delivery of the verification code, before requesting a new one. If you try two
```
Comment (rebamrich) [inline source/help/registerhelp.md:44]: No need for comma here.

**Review by rebamrich — APPROVED** [review]
I made some suggestions. Otherwise looks good!

File: source/help/policies/identity_and_affiliation.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    reason, is possible grounds for immediate and permanent suspension.
+-   Users must comply with arXiv's [Code of conduct](code_of_conduct.md).
+
+## Managing your identity
+
+arXiv recommends a consistent representation of your name to aid search and discovery of your works. 
+
+Linking your ORCID to your arXiv account is recommended to connect your arXiv papers to your scholarly record.
+
+![ORCID logo](../../assets/orcid-logo.png){.margin5 align="left" width="20" role="presentation"}
```
[the remark this answers is not in the document]
Comment (jimentwood) [reply source/help/policies/identity_and_affiliation.md:22]: I think that is just a template issue.
When it renders in the arXiv template Preview it looks OK:
https://storage.googleapis.com/arxiv-docs-prs/ET-35_identity/help/policies/identity_and_affiliation.html

**Review by abartley829 — APPROVED** [review]
Looks good

**Pull request merged**